src/database/prisma/schema.prisma

```diff
@@ -33,6 +33,9 @@ enum DocumentType {
     GUIDELINE
     MEETING
     OTHER
+    PLAN_OF_OPERATIONS
+    FRAMEWORK_BUDGET
+    STRATEGIC_GOALS
```

Comment:
I was given a took these names from a Swedish-English wordlist I was given. So if they are used elsewhere, it might be a good idea to keep as is for consistency?